source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,485 @@
+
+   .. tab:: Create an App Service
+      :tabid: app-service
+
+      Refer to the documentation to :ref:`create-app`.
+
+      When you choose your App type, select ``Edge Server`` or ``Device Sync``. 
+
+      .. include:: /includes/edge-server-enable-device-sync.rst
+
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server supports a subset of authentication providers:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`custom-jwt-authentication`
+- :ref:`api-key-authentication`
+
+You must enable at least one of these providers to enable clients to connect
+to the Edge Server.
+
+.. _create-edge-server-instance:
+
+Create an Edge Server Instance
+------------------------------
+
+After creating an Edge Server Service, you can add Edge Servers to it. The
+service can handle multiple Edge Servers, each of which has its own 
+registration key.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Add a server
+
+            Click the :guilabel:`Add a server` button to walk through the 
+            process of configuring your new Edge Server instance.
+
+            If this is the first server in your service, you're prompted to 
+            select a cluster, connect a data source, and optionally specify 
+            which collections to sync to the Edge Server. This configuration
+            applies to all Edge Servers in this service.
+
+            For subsequent servers, you're prompted to name you Edge Server.
+
+            The UI provides an installation script, which you can copy and 
+            paste into the terminal on the device where you want to install 
+            the Edge Server. You also get a token, which you will use in the 
+            when you configure the Edge Server instance.
+
+         .. step:: Install and Configure the Edge Server code
+
+            Copy and paste the installation script into the terminal on the
+            device where you want to install the Edge Server. For more details,
+            refer to :ref:`edge-server-install-and-configure`.
+
+            After you configure and start the Edge Server, its 
+            :guilabel:`Health` status in the Edge Server dashboard changes
+            from :guilabel:`Pending` to :guilabel:`Active`.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Call the Create Edge Server Endpoint
+
+            Call the <CREATE EDGE SERVER> endpoint to add an Edge Server 
+            instance to your service. You must provide the the App's internal 
+            ``appId`` hex string:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            This returns <INSERT RETURN DETAILS HERE>.
+
+         .. step:: Install the Edge Server code
```

Comment:
Oh I don't think the admin endpoint returns the install instructions.

Reply:
The other page that I'm linking out to here has the install instructions. I've added those in PR #733 .